### ISSUE TITLE
Is there more pre-trained models?

### ISSUE THREAD

**Issue by seniusen** [JacobYuan7/RLIPv2#5]
Thanks for your excellent work!

I'm wondering if more pre-trained models will be released, such as the R-Tagger based on SwinL and the RLIP-ParSeD series.

**Comment by JacobYuan7** [comment]
@seniusen 
Hi, the R-Tagger based on Swin-L is not used by default. I find one interesting thing about R-Tagger is that Swin-L does not improve the performance much. Therefore, I keep using the relatively efficient version of R-Tagger (R50). 

Is there a particular reason for you to use RLIP-ParSeD series pre-training models? I thought RLIP-ParSeDA performs better while maintaining similar inference speed. Thus, I only release RLIP-ParSeDA.

**Comment by seniusen** [comment]
@JacobYuan7 
Thank you for your reply. I just want to reproduce your results. Since then, I'll choose RLIP-ParSeDA.

Another question. In your paper, Table 4 shows that RLIPv2-ParSeD initialized from R-Tagger is slightly superior to the model initialized from COCO. While in Sec.5 Implementation details, I see RLIPv2 is initialized from COCO detection parameters. Am I understand right?

Or as you say, RLIPv2 based on R50 is initialized from R-Tagger, while RLIPv2 based on Swin-T and Swin-L is initialized from COCO.

**Comment by JacobYuan7** [comment]
@seniusen 
Hi. Thanks for pointing out. I think there might be a misunderstanding. In Table 4, the default initialization for COCO leads to 13.62, while the result for R-Tagger initiailization achieves 13.39. Therefore, the previous one is better. Btw, the aim of table 4 is not to compare the effectiveness of initialization for RLIPv2 but to verify the necessity of denoising pre-training and attention masks for R-Tagger. The initiailization for RLIPv2 is object detection parameters as indicated in **Implementation details**.

**Comment by seniusen** [comment]
@JacobYuan7 
I may have made a mistake in comparing the rare mAP in Table 4. Now everything is clear. Thank you!
